src/test/groovy/com/palantir/gradle/versions/VersionsPropsPluginIntegrationSpec.groovy

```diff
@@ -18,19 +18,29 @@ package com.palantir.gradle.versions
 
 import groovy.util.slurpersupport.GPathResult
 import groovy.util.slurpersupport.NodeChildren
-import nebula.test.IntegrationTestKitSpec
 
-class VersionsPropsPluginIntegrationSpec extends IntegrationTestKitSpec {
+class VersionsPropsPluginIntegrationSpec extends IntegrationSpec {
```

Comment:
still really unhappy that we're not using nebula-test as IntelliJ test running consistently uses the wrong code (cc @ferozco)